### PR TITLE
Added initial get challenge task by id

### PR DESCRIPTION
**Pull request by TheHollidayInn**
These are for the "tests for routes used to manage tasks for challenges" item in #6776 .

### CODE REVIEW

**Comment by paglias** [comment]
@TheHollidayInn I noticed the get tasks for user tests are under at /tasks/GET-tasks_user.test.js, let's move these ones to /tasks/GET-tasks_challenge_challengeId.test?

**Comment by paglias** [comment]
I'd add a test to check that a wrong id is handled correctly then I think we can merge!

File: website/src/controllers/api-v3/tasks.js [hunk 1]
```diff
@@ -257,7 +257,7 @@ api.getTask = {
     if (!task) {
       throw new NotFound(res.t('taskNotFound'));
     } else if (!task.userId) { // If the task belongs to a challenge make sure the user has rights
-      let challenge = await Challenge.find().selec({_id: task.challenge.id}).select('leader').exec();
+      let challenge = await Challenge.find({_id: task.challenge.id}).select('leader').exec();
```
Comment (paglias) [inline website/src/controllers/api-v3/tasks.js:260]: ops...! thanks for catching this

**Comment by TheHollidayInn** [comment]
Are you saying to move the tests out of the challenges directory and rename the file to `/tasks/GET-tasks_challenge_challengeId.test` ?

And, I'll get that test added!

**Pull request merged**